docs/source/developer/interactives.rst

```diff
@@ -1,126 +1,163 @@
+ 
+.. note::
 
-JavaScript Files
+  When creating a new interactive, we suggest you copy the ``static/interactives/example-structure/`` directory and rename it to the slug for your interactive.
+  This directory has the basic structure already mapped out to get you started.
+
+  Similarly, within the ``templates/interactives/`` directory, there is a starter template for each of the interactive types (``iframe-template.html``, ``in-page-template.html`` and ``whole-page-template.html``).
+
+Interactive Requirements
 ------------------------------------------------------------------------------
 
-- `third-party/materialize.min.js` - Event handling for MaterializeCSS framework.
-- `third-party/foundation.min.js` - Event handling for Foundation framework.
-- `third-party/jquery.js` - A fast, small, and feature-rich JavaScript library. It makes things like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API that works across a multitude of browsers. Required by MaterilizeCSS and Foundation frameworks.
-- `third-party/codemirror.js` - Logic required to create and use [CodeMirror](https://codemirror.net/) editors.
-- `third-party/codemirror-mode-regex.js` - Logic required to highlight text with regular expression syntax in CodeMirror editors.
-- `third-party/es5-shim.min.js` - ECMAScript 5 compatibility shims for legacy (and modern) JavaScript engines.
-- `third-party/es6-shim.min.js` - ECMAScript 6 compatibility shims for legacy (and modern) JavaScript engines.
-- `third-party/iframeResizer.contentWindow.min.js` - Required [iFrame Resizer](http://davidjbradshaw.github.io/iframe-resizer/) file for interactives to be used with `iframe` mode. This *child* file is used within the content window, and talks to the *parent* file on the page, do automatically resize the iFrame window to fit the content.
-- `third-party/jsencrypt.js` - [JSEncrypt](http://travistidwell.com/jsencrypt/) is a Javascript library to perform OpenSSL RSA Encryption, Decryption, and Key Generation.
-- `third-party/modernizr.js` - [Modernizr](https://modernizr.com/) is a small piece of JavaScript code that automatically detects the availability of next-generation web technologies in your user's browsers. Rather than blacklisting entire ranges of browsers based on “UA sniffing,” Modernizr uses feature detection to allow you to easily tailor your user's experiences based on the actual capabilities of their browser.
-- `third-party/nouislider.min.js` - Event handling for the [noUiSlider library](http://refreshless.com/nouislider/), used within many colour interactives.
-- `third-party/wNumb.js` - [wNumb](http://refreshless.com/wnumb/) is a number and money formatting library. Can be used independently, or with noUiSlider to format labels.
+A completed interactive requires the following to be included into the repository for release:
+
+- The interactive is primarily written in JS, HTML, and CSS.
```

Comment:
Why is this a requirement?

Reply:
this was copied from the original interactive docs, and is probably in need of a discussion! have removed it for now.